app/components/views/Bond/index.tsx

```diff
@@ -593,44 +752,48 @@ export const Bond: FC<Props> = (props) => {
                   Number(bondState?.bondQuote) > Number(bondState?.maxBondPrice)
                 }
                 value={trimWithPlaceholder(bondState?.maxBondPrice, 2, locale)}
```

Comment:
As mentioned in discord, this needs to fallback to capacity, if the capacity is lower than the maxbondprice.